### ISSUE TITLE
When autofilling a search term on info page, the wrong background is selected in the form field

### ISSUE THREAD

**Issue by tsteur** [piwik/piwik#18857]
This happens eg on Chrome when selecting an entry from the historical autofill list.
<img width="961" alt="image" src="https://user-images.githubusercontent.com/273120/155916727-40f88511-a2fa-4789-a347-24f920dd6ed7.png">

Then the background color changes

<img width="916" alt="image" src="https://user-images.githubusercontent.com/273120/155916681-1e501aee-cd32-4f04-8a32-d0bbab71cc7e.png">

because of this CSS

<img width="331" alt="image" src="https://user-images.githubusercontent.com/273120/155916786-75aa76b2-2381-4348-b122-40a7fae694da.png">

**Comment by dideldumthecat** [comment]
TL;DR: I would move to close this issue. Here's why:

* This is Chrome's way of highlighting fields that were auto-filled by the browser, so that the user is alerted to this fact and can double-check the input. I see it on login fields and payment forms with my password manager as well. See https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Selectors/:autofill
* The current color contrasts work in Matomo's light and dark modes.

Since removing this styling is not recommended, the only alternative would be to adjust the colors to more closely match Matomo's look and feel — but I'm not sure that deviating from the default appearance would add much value to the user experience. Also, as of 2026, Firefox doesn't support this.

FYI: To reproduce it, I had to use an extension, "Form Recover", because it seems that by default Chrome no longer keeps a history for search fields.
